### DISCUSSION TITLE
Mantine modals manager: new type for confirm and cancel labels?

### DISCUSSION THREAD

**Discussion by dshmatok** [mantinedev/mantine#1286] [Feature requests]
Hi! First of all, thanks for the amazing Mantine library! :)
I want to aks: is it possible replace for `confirm` and `cancel` labels type `string` to `React.ReactNode`? 

https://github.com/mantinedev/mantine/blob/5d5c2d3648f8530b6d5fe58b96a298da2bf80f42/src/mantine-modals/src/context.ts#L7

The contents of these labels are placed into `button` elements and according to the specification, not only `string` can be placed there.

https://github.com/mantinedev/mantine/blob/5d5c2d3648f8530b6d5fe58b96a298da2bf80f42/src/mantine-modals/src/ConfirmModal.tsx#L50-L52

https://github.com/mantinedev/mantine/blob/5d5c2d3648f8530b6d5fe58b96a298da2bf80f42/src/mantine-modals/src/ConfirmModal.tsx#L54-L57

Thanks!

**Comment by rtivital** [comment]
Yeah, I think it is possible, feel free to submit a PR for this

**Reply by dshmatok** [reply]
Thanks! https://github.com/mantinedev/mantine/pull/1287
